src/util/pg.py

```diff
@@ -608,18 +608,45 @@ def view_exists(cr, view):
     return bool(cr.rowcount)
 
 
-def remove_constraint(cr, table, name, cascade=False):
+def remove_constraint(cr, table, name, cascade=False, warn=True):
+    """
+    Remove a table constraint.
+
+    This function removes the constraint `name` from the table named `table`. It also
+    removes records from `ir_model_constraint` and its xmlids. It logs not found
+    constraints.
+
+    .. note::
+
+       If there is no constraint `name`, this function will attempt to remove
+       ``{table}_{name}``, the latter is the default name the ORM uses for constraints
+       created from `_sql_constraints`. This name mismatch will be logged.
```

Comment:
```suggestion
       If there is no constraint `name` on `table`, this function will attempt to remove
       ``{table}_{name}``, the latter is the default name the ORM uses for constraints
       created from `_sql_constraints`.
```

I wouldn't add the last ambiguous part to this note. In fact, mismatches are logged iff `name` is not found in the first place.

Reply:
I removed it, although it was _not_ ambiguous: name mismatches are always logged, in the form "... not found in ..." ;)